pkg/manager/suspender/suspender.go

```diff
@@ -223,6 +223,12 @@ func needsSuspendComponent(cluster v1alpha1.Cluster, comp v1alpha1.MemberType) b
 
 // canSuspendComponent checks whether suspender can start to suspend the component
 func canSuspendComponent(cluster v1alpha1.Cluster, comp v1alpha1.MemberType) (bool, string) {
+	// only support to suspend Normal or Suspend cluster
+	// If the cluster is Upgrading or Scaling, the sts can not be deleted.
+	if !cluster.ComponentIsNormal(comp) && !cluster.ComponentIsSuspending(comp) {
+		return false, "component phase is not Normal or Suspend"
```

Comment:
```suggestion
		return false, "component phase is not Normal or Suspending"
```

?

Reply:
the value of `SuspendPhase` is "Suspend" because it contain suspended status and suspending status. So this is "Suspend"